stdlib/sql/hdb.go

```diff
@@ -209,22 +209,32 @@ END;
 
 // hdbAddIfNotExist adds SAP HANA specific table existence check to CREATE TABLE statement.
 func hdbAddIfNotExist(table string, query string) string {
-	var where string
+	var where, schema, tbl string
 	var args []interface{}
-	parts := strings.SplitN(strings.ToUpper(table), ".", 2) // schema and table name assumed uppercase in HDB by default (see Notes)
-	if len(parts) == 2 {                                    // fully-qualified table name
+	// Schema and table name assumed uppercase in HDB by default (see Notes)
+	// XXX: since we are currently forcing identifiers to be UPPER CASE.
+	//  Shadowing the table param ensures we use the UPPER CASE form regardless
```

Comment:
In the original impl, the `CREATE TABLE` DDL would include the input string as-is if the schema wasn't specified (a bug that surfaced when we started quoting all identifiers).